content/docs/intro/concepts/runtime-functions.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Runtime Functions"
```

Comment:
I never loved this name for this section - but it stands out even more now that it's top level.  These are really just a grab bag of things that weren't covered in other sections.  I don't have any great recommendations for these - but this is neither a list of all SDK helper functions, nor any specific meaningful grouping of functionality.  Thoughts on alternative names?

Reply:
I re-labeled this title as "Pulumi SDK Helper Functions" and put it in the ref section. It feels like we can then add to this topic as need to capture all of the helper functions available.

Reply:
I don't think this quite works in the new form.  These helpers are already documented in the API reference section, and the style and scope of this content doesn't really "fit" in the new location.  Looking at the content there - here's a suggestion:
1. Move the "Getting the Current Project or Stack" content into the Project and Stack sections (if it's not already there)
2. Move the Logging section into a new top level section in Architecture and Concepts.  In the future, we can expand this and/or collapse into some other section if needed.
3. Move the "Serializing Lambdas" section into a new top level section named "Function Serialization" in Architecture and Concepts.  Combine the contents of the Serializing JavaScript Functions page it links to inline into the page.

Reply:
I implemented all three changes in https://github.com/pulumi/docs/pull/4707/commits/9b8bbfadbcf53b17ea26e4ef3afc56a7ee028471; however, item 2 is one that feels the most out of place in the concepts section (as logging isn't really a "concept"). Once we get a proper auditing/logging/monitoring section in the docs, then I'll move this content over to that location.